pkg/logs/build_logs.go

```diff
@@ -0,0 +1,234 @@
+package logs
```

Comment:
Originally the "logs/waiting" functionality was added to kpack so that it could be used by clients of kpack to get the functionality. (e.g. concourse-kpack-resource) Perhaps it makes sense to move it to the kp-cli and begin treating the kp cli as a library. (Or maybe a kpack-client repo 😍?) but, I wonder if we can keep the logs to reduce the size of this PR and discuss those merits on their own?